website/source/docs/commands/apply.html.markdown

```diff
@@ -53,10 +53,11 @@ The command-line flags are all optional. The list of available flags are:
   multiple times.
 
 * `-var 'foo=bar'` - Set a variable in the Terraform configuration. This
-  flag can be set multiple times.
+  flag can be set multiple times. Variable values are interpreted as HCL, so
```

Comment:
Reworked to mention HCL on the syntax page and link off to it every time it is mentioned.